server/api/routers/profile.ts

```diff
@@ -0,0 +1,46 @@
+import { createTRPCRouter, protectedProcedure } from "server/api/trpc";
+import { ProfileSchema } from "@lib/schemas/profile";
+import { z } from "zod";
+
+export const profileRouter = createTRPCRouter({
+  update: protectedProcedure
+    .input(ProfileSchema)
+    .mutation(async ({ input, ctx }) => {
+      // Update phases and questionnaires
+
+      await ctx.db.profile.update({
+        where: {
+          userId: input.userId,
```

Comment:
Prisma will throw an error here because you're not using the PK here. So just adding an "id" here would be fine (see Prisma Schema for "id" under the Profile Model)